### PR TITLE
Adds thermo-regulation directly into Cryo.

### PR DESCRIPTION
**Pull request by 9600bauds**
When a patient is under 31C, ejecting them from Cryo will take 4 seconds, during which their temperature will be regulated to 34.5C (meaning they will only shiver for a little while).
Adds pressurehiss.ogg, which is played when said cycle starts.
Added an "Occupant body temperature" reading to the Cryo UI. It becomes green when the patient's body temp is low enough for Cryoxadone and Clonexadone to work.
Webbum: http://a.pomf.cat/daqdmz.webm (splash now lasts a little longer than that)

Addresses part of #8047
Intended to go along with #8194
This is also a discussion on whether this ejection delay should be present or not. I think this should be left to player choice.

### CODE REVIEW

**Comment by ihadtoregisterforthis** [comment]
:cry:

**Comment by CptWad** [comment]
>2 second delay

I dont see a problem with that. My only worry is what it does for self ejection when you get left in cryo _forever_

**Comment by Zth--** [comment]
It feels like it's too fast. I would do it with 4 or maybe 5 seconds

**Comment by 9600bauds** [comment]
I'm of the opinion that it used to be 0 seconds, and nobody thought it was too short back then.
Regardless, it's already 3 seconds.

Also, self-ejection works the same. After the 30 seconds of wait, the 3 second cycle starts, so you have to wait 33 seconds I guess

**Comment by Imagebard** [comment]
:+1:

**Comment by Exxion** [comment]
You know, I do legitimately like the Mancrowave (which isn't called that but its full name is too long so fuck it), but not enough to argue for it again or watch the inevitable arguments about it in the thread
So I guess I can't really complain about anything that ends those arguments

**Comment by ghost** [comment]
It'd be a shame to lose the ability to mask plasmamen or vox safely when ejecting, that was one of the changes I know was fairly popular. Could you make an exception for them? Otherwise, I'll take this as the price of peace and use the THR elsewhere.

**Comment by CptWad** [comment]
THR as artifact for xenoarch when

**Comment by LordMyroc** [comment]
You could always add an option to override thermal regulation and eject patients at their current temperature if vox and plasmamen are an issue. Or alternatively, just make everyone not breathe for a few seconds after exiting cryo.

**Comment by 9600bauds** [comment]
I'll be looking into making people not breathe after Cryo. Ejecting people at their current temperature would be more of an emag thing, however, since it would kill them.

**Comment by Kurfursten** [comment]
> Cell temp in K
> Body temp in C

AAAAAAAAAAAAAAAAAA

**Comment by 9600bauds** [comment]
Body temp in K would be extremely inconvenient, and I'm not sure if cell temp in C would be an improvement

**Comment by Zth--** [comment]
Change everything to C. Fuck kelvin

**Comment by CptWad** [comment]
but I like kelvin

**Comment by Exxion** [comment]
Fuck anything other than SI units. Kelvin all the way.

**Comment by 9600bauds** [comment]
Added period of grace for filthy non-oxygen-breathers

File: html/changelogs/9600bauds_chilidog.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+changes: 
+- rscadd: Cryo cells now have a short regulation cycle when ejecting a hypothermic occupant, which brings their temperature back to safe levels. This is intended to bring it back to it's classic functioning, where it ejected patients at safe temperatures anyways.
```
Comment (ghost) [inline html/changelogs/9600bauds_chilidog.yml]: Its, not it's, and cryo always ejected people at a cold temperature, which in fact was raised with hypothermia

**Comment by ghost** [comment]
Eh, I'd make the time a bit longer but this is fine.

File: html/changelogs/9600bauds_chilidog.yml (hunk 1 above)
Comment (9600bauds) [inline html/changelogs/9600bauds_chilidog.yml]: The big overlooked detail here is that said cold temperature was perfectly safe pre-hypothermia.

**Comment by 9600bauds** [comment]
Upped the time to 4 secs given that everyone said it was kind of short.

File: html/changelogs/9600bauds_chilidog.yml (hunk 1 above)
Comment (ghost) [inline html/changelogs/9600bauds_chilidog.yml]: And the new cold temperature was perfectly safe post hypothermia.

Comment (9600bauds) [inline html/changelogs/9600bauds_chilidog.yml]: Stacking oxyloss, maybe not lethal on it's own but certainly not safe.

Comment (ghost) [inline html/changelogs/9600bauds_chilidog.yml]: I'd reword this altogether anyway, it's an incredibly convoluted way of saying "cryo now pops you out at 31C".

**Comment by ghost** [comment]
:+1:

**Comment by NigglyWiggly** [comment]
Change everything to Fahrenheit.
We're an **AMERICAN** station, after all.

**Comment by Exxion** [comment]
The server is hosted in Canada though

**Comment by ghost** [comment]
Fahrenheit is awful.

**Comment by Exxion** [comment]
> 0 is the coldest temperature he could achieve in his lab, despite it getting colder than that outside in many places
> Other defining point was supposed to be based on human body temperature but he fucked up

I'm not even sure Fahrenheit was a good _scientist_

**Comment by ihadtoregisterforthis** [comment]
imperial system explained

**Comment by Exxion** [comment]
Hydrogen actually weighs 1.008 grams per mole but close enough I guess

**Comment by ghost** [comment]
700px-countries_that_use_fahrenheit svg

**Comment by ghost** [comment]
Countries that use Fahrenheit.

**Comment by clusterfack** [comment]
The system literally based on the boiling point of water is better at calculating the energy to boil water.

Wew lad

**Comment by PJB3005** [comment]
@clusterfack that's just an example, the SI / metric units actually have some fucking sense behind them.

**Comment by clusterfack** [comment]
Not really, now if they used the triple points of two pure elements as the scale and absolute zero as the zero point, then they would have some sense behind them.

**Comment by NigglyWiggly** [comment]
**LOOK AT ALL THESE FUCKING COMMIES**

FUCK OFF BACK TO YOUR UNAMERICAN SPACE STATIONS WE USE OUR SOPHISTICATED AND VERY REASONABLE, NOT STUPID AND COMPLETELY FUCKED UP SYSTEMS OF MEASUREMENT HERE.

ᴵ'ᵐ ˢᵒʳʳʸ ᶠᵒʳ ᵈᵉʳᵃᶦᶫᶦᶰᵍ ᵗʰᶦˢ ᶜᵒᶰᵛᵉʳˢᵃᵗᶦᵒᶰ

**Comment by gbasood** [comment]
> celt gave the thumbs up

WELL THEN TIME  TO MERGE
